raiden/network/rpc/client.py

```diff
@@ -726,6 +752,7 @@ def filter_changes(self, fid):
             for c in changes
         ]
 
+    @_checkNodeConnection
```

Comment:
what about the transactions?

Reply:
Good point!

Reply:
Btw, I wonder if the transaction pool is actually persisted, if that is the case the retries may fail like with a known transaction error: https://github.com/raiden-network/raiden/issues/1061

Reply:
Apparently it is persistent for ones own transactions.

Reply:
So, it raises the exception with the message `known transaction`? We need some way to tell if that's bad or good, and handle errors like the one in the PR.

If you want we can merge this and look at the repeated transaction later.